src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -1910,5 +1917,170 @@ public void testCanHandleSymbollicAlleleFuncotations() {
+                    "." + outputFormatType.toString().toLowerCase();
+            final File outputFile = createTempFile(tmpOutDir + File.separator + inputVcfName + ".funcotator", "." + outputFormatType.toString().toLowerCase());
+
+            final ArgumentsBuilder arguments = createBaselineArgumentsForFuncotator(
+                    inputVcfName,
+                    outputFile,
+                    referencePath,
+                    dataSourcesPath,
+                    referenceVersion,
+                    outputFormatType,
+                    true);
+            runCommandLine(arguments);
+        }
+
+    }
+
+    /** In this test, we are giving as input a vcf which has already been annotated.
+     * Because we are not giving the flag argument which indicates that we should reannotate,
+     * then the default argument will be used which is to no reannotate and instead to throw
+     * an error.
+     */
+    @Test(dataProvider = "provideForCheckIfAlreadyAnnotatedTestWrong", expectedExceptions = UserException.BadInput.class)
+    public void testCheckWhenOverrideReannotate(final String inputVcfName,
+                                                           final String referencePath,
+                                                           final String referenceVersion,
+                                                           final String dataSourcesPath,
+                                                           final String expectedOutputPath)
+    {
+
+        for ( final FuncotatorArgumentDefinitions.OutputFormatType outputFormatType :
+                FuncotatorArgumentDefinitions.OutputFormatType.values()) {
+            if (outputFormatType.equals(FuncotatorArgumentDefinitions.OutputFormatType.SEG)) {
+                continue;
+            }
+            final String typeCorrectedExpectedOutPath = FilenameUtils.removeExtension(expectedOutputPath) +
+                    "." + outputFormatType.toString().toLowerCase();
+            final File outputFile = createTempFile(tmpOutDir + File.separator + inputVcfName + ".funcotator", "." + outputFormatType.toString().toLowerCase());
+
+            final ArgumentsBuilder arguments = createBaselineArgumentsForFuncotator(
+                    inputVcfName,
+                    outputFile,
+                    referencePath,
+                    dataSourcesPath,
+                    referenceVersion,
+                    outputFormatType,
+                    true);
+            runCommandLine(arguments);
+        }
+
+    }
+
+    /** In this instance, we are again giving an already annotated file, but
+     * we are using the flag argument "true" to indicate that we want this file
+     * to be reannotated. There should be no errors.
+     */
+    @Test(dataProvider = "provideForCheckIfAlreadyAnnotatedTestWrong")
+    public void testCheckWhenWantReannotate(final String inputVcfName,
```

Comment:
The names of your test methods are a bit hard to understand -- I suggest tweaking the names to make it clearer what configuration each method is running with. Eg.,

`testAlreadyAnnotatedInputWithOverrideArgument()`
`testAlreadyAnnotatedInputWithoutOverrideArgument()`